buildscripts/azure-pipelines-jobs-windows.yml

```diff
@@ -68,6 +68,24 @@ jobs:
   - bash: pushd builds && rm -rf tests *.pdb && popd
     displayName: 'Clean up Artifacts'
   - publish: builds/
-    displayName: 'Publish Artifacts'
+    displayName: 'Publish ThinApp'
     condition: and(succeeded(), eq(variables['configuration'], 'Release'))
     artifact: ${{ parameters.name }}$(configuration)
+  - task: DownloadSecureFile@1
+    name: makensis
+    condition: and(succeeded(), eq(variables['configuration'], 'Release'))
+    displayName: 'Download NSIS'
```

Comment:
This is already on the build machines, I assume it's in the default location but I'm not sure if it's 32 or 64 bit.